docs/case-studies/gitops.md

```diff
@@ -110,67 +252,128 @@ spec:
         commitMessage: "Image: {{range .Updated.Images}}{{println .}}{{end}}"
 ```
 
-Apply the file to the cluster and check the `Application` in clusters, we can see that the `git-app` automatically pulls the config from Git Repository and apply the application to the cluster:
+Apply the above file to the clusters, we can see that the application in `apps/` is automatically deployed:
 
 ```shell
-$ vela ls
-
-APP                	COMPONENT     	TYPE      	TRAITS 	PHASE  	HEALTHY	STATUS	CREATED-TIME
-first-vela-workflow	test-server	    webservice	ingress	running	healthy	      	2021-09-10 11:23:34 +0800 CST
-git-app            	gitops        	kustomize 	       	running	healthy	      	2021-09-10 11:23:32 +0800 CST
+APP   	COMPONENT       	TYPE      	TRAITS 	PHASE  	HEALTHY	STATUS	CREATED-TIME
+apps  	apps            	kustomize 	       	running	healthy	      	2021-09-27 16:55:53 +0800 CST
+infra 	database-config 	kustomize 	       	running	healthy	      	2021-09-26 20:48:09 +0800 CST
+my-app	my-server       	webservice	ingress	running	healthy	      	2021-09-27 16:55:55 +0800 CST
+mysql 	mysql-controller	helm      	       	running	healthy	      	2021-09-26 20:48:11 +0800 CST
+└─  	mysql-cluster   	raw       	       	running	healthy	      	2021-09-26 20:48:11 +0800 CST
 ```
 
-We can `curl` the `Ingress` to see the current version in code:
+Curl the Ingress of the app, we can see that the current version is `0.1.5` and the application is connected to the database successfully:
 
-```shell 
-$ curl -H "Host:testsvc.example.com" http://<your-ip>
+```shell
+$ kubectl get ingress
+NAME        CLASS    HOSTS                 ADDRESS         PORTS   AGE
+my-server   <none>   testsvc.example.com   <ingress-ip>    80      162m
+
+$ curl -H "Host:testsvc.example.com" http://<ingress-ip>
 Version: 0.1.5
+
+$ curl -H "Host:testsvc.example.com" http://<ingress-ip>/db
+User: KubeVela
+Description: It's a test user
+```
+
+## Modify the configuration
+
+After the first deployment, we can modify the files in config repo to update the applications in the cluster.
+
+Modify the domain of the application's Ingress:
+
+```yaml
+...
+      traits:
+        - type: ingress
+          properties:
+            domain: kubevela.example.com
+            http:
+              /: 8089
+```
+
+Check the Ingress in cluster after a while:
```

Comment:
Who is  listening to the config repo and apply to the cluster?
IIRC the previous sections have only mentioned how to make KubeVela committing to the Code Repo